src/aphroditeInterface.js

```diff
@@ -1,4 +1,4 @@
-import aphrodite from 'aphrodite';
+import { StyleSheet, css, flushToStyleTag } from 'aphrodite';
```

Comment:
Is this difference mandatory or just ease of reading?

Reply:
It is mandatory

Reply:
Alternatively, I think we could `import *` but that's not great.